charts/cp-ksql-server/templates/deployment.yaml

```diff
@@ -81,15 +81,10 @@ spec:
             value: {{ template "cp-ksql-server.serviceId" . }}
           - name: KSQL_KSQL_SCHEMA_REGISTRY_URL
             value: {{ template "cp-ksql-server.cp-schema-registry.service-name" . }}
+          - name: KSQL_KSQL_CONNECT_URL
+            value: {{ template "cp-ksql-server.cp-kafka-connect.service-name" . }}
           - name: KSQL_HEAP_OPTS
             value: "{{ .Values.heapOptions }}"
-          {{- if .Values.ksql.headless }}
```

Comment:
why you removed this 2 values? Everything else looks fine

Reply:
Hi @im-pratham,
I removed these two values in an other recent commit '[update ksql and fix helm upgrade](https://github.com/Hamdiovish/cp-helm-charts/commit/027baf4213837601b0c4c2887de6544fc40646bc)', the problem with these values is that it cause a problem with the command "helm upgrade", i don't remember exactly whether the problem is related to Helm or to Kafka, but i was able to process the "helm upgrade" after removing these two values.